src/sortStudents.ts

```diff
@@ -1,16 +1,75 @@
-
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade'
+}
+
+export enum SortOrder {
+  Asc = 'asc',
+  Desc = 'desc'
+}
+
+function getAveregeGrade(student: Student): number {
+  return student.grades
+    .reduce((sum, grade) => sum + grade, 0) / student.grades.length;
 }
 
-// create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const copiedStudents: Student[] = [...students];
+
+  switch (sortBy) {
+    case SortType.Name:
+    case SortType.Surname:
+      return (order === SortOrder.Asc)
+        ? copiedStudents.sort((student1, student2) => {
+          return student1[sortBy].localeCompare(student2[sortBy]);
+        })
```

Comment:
avoid return where it's possible

```suggestion
        ? copiedStudents.sort((student1, student2) => (
           student1[sortBy].localeCompare(student2[sortBy]);
        ))
```